src/pwm.rs

```diff
@@ -80,6 +81,26 @@ pub trait Pins<REMAP, P> {
     const C3: bool = false;
     const C4: bool = false;
     type Channels;
+
+    fn check_used(c: Channel) -> Channel {
+        if (c == Channel::C1 && Self::C1)
+            || (c == Channel::C2 && Self::C2)
+            || (c == Channel::C3 && Self::C3)
+            || (c == Channel::C4 && Self::C4)
+        {
+            c
+        } else {
+            panic!("Unused channel")
+        }
```

Comment:
@TheZoq2 @TeXitoi 
In fact this code must be optimized by compiler. So no panic will be in code if user specify right channel.

Reply:
Ideally I want to avoid panics if the user specifies the wrong channel as well :) But I don't see a great way to do that at the moment